test/functional/feature_evm_dst20.py

```diff
@@ -937,7 +937,7 @@ def run_test(self):
 
         # Contract ABI
         self.abi = open(
```

Comment:
Can we leave a short comment to indicate the rationale to use dst20 facade as the abi, but v1 as the bytecode ?

Reply:
Done: d5d3839dbcac45af568d3df3fb28c1064504f32b